tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +83,16 @@ func (tt *tester) runLoop() {
 			plog.Printf("[round#%d case#%d] recovered failure", i, j)
 
 			if !tt.consistencyCheck {
+				plog.Printf("[round#%d case#%d] canceling the stressers...", i, j)
```

Comment:
I think this mean we need to add more ignore errors to here

https://github.com/coreos/etcd/blob/master/tools/functional-tester/etcd-tester/stresser.go#L96